public/data/itzwow.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "AMIT KUMAR BERA",
+    "type": "personal",
+    "bio": "Web Developer, Open Source Enthusiast! & Explorer",
+    "avatar": "https://github.com/itzwow.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/itzwow",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/itzyourab",
+        "icon": "twitter"
+      },{
+        "name": "Follow me on Linkedin",
+        "url": "https://www.linkedin.com/in/amit-kumar-bera-990954205/",
```

```suggestion
        "url": "https://www.linkedin.com/in/amit-kumar-bera-990954205",
```

The / is not needed in the url, it works anyway. It just gets added when you copy the url from the taskbar.

---

As you mentioned this will still work, I think these types of feedback are unnecessary notifications